### PR TITLE
General improvements [Ready To Merge]

### PR DESCRIPTION
**Pull request by anar4732**
- Added a better eln to other convertor texture.
- Added marcin212's MineTweaker support.
I made minor bug fixes.
It works very well.

`import minetweaker.item.IIngredient;`
`import mods.electricalage.Macerator;`
` `
`Macerator.addRecipe(<minecraft:wheat> as IIngredient, 1000.0, [<MyMod:itemA>, <MyMod:itemD>]);`

#307

### CODE REVIEW

**Comment by anar4732** [comment]
ready to merge @cm0x4D

**Comment by cm0x4D** [comment]
Sorry, I am pretty busy the next weeks. @Baughn could you have a look at?

File: src/main/java/mods/eln/integration/minetweaker/MinetweakerIntegration.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import mods.eln.integration.minetweaker.machines.Compressor;
+import mods.eln.integration.minetweaker.machines.Macerator;
+import mods.eln.integration.minetweaker.machines.Magnetizer;
+import mods.eln.integration.minetweaker.machines.PlateMachine;
+
+public class MinetweakerIntegration {
+	public static MinetweakerIntegration instance = new MinetweakerIntegration();
+
+	
+	public void initialize(){
```
Comment (cm0x4D) [inline src/main/java/mods/eln/integration/minetweaker/MinetweakerIntegration.java:13]: Could be static? If yes, there would be no need for a singleton...

**Review by cm0x4D — COMMENTED** [review]
LGTM - tested only one recipe with MineTweaker and it worked as expected.

The new exporter texture is nice too.

**Review by cm0x4D — APPROVED** [review]

**Comment by DonBruce64** [comment]
Heads up: Minetweaker support for removal does not work with any Recipe for the Plate Machine, Compressor, or Magnatizer.  Not sure why, as I can add recipes just fine, and remove recipes from the Macerator just fine too.

**Comment by bloxgate** [comment]
I'm planning on implementing some sort of minetweaker support in with #652, which I've resumed work on recently. I'm not sure how that'll end up affecting this PR.

**Pull request closed without merging**